css/style.css

```diff
@@ -411,13 +719,175 @@
     padding-right: 35px;
   }
 
+  .data__figures {
+    font-weight: 600;
+  }
+
   .test-btn {
     padding-right: 0;
   }
 
   .testdrive {
     margin-right: 60px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+
+  .wrap-information, .wrap {
+    height: 747px;
+    }
+
+  .red-first {
+    width: 400px;
+    height: 100%;
+    padding: 37px 47px 0 45px;
+  }
+
+  .description__title {
+    font-size: 25px;
+  }
+
+  .drive {
+    padding-bottom: 60px;
+  }
+
+  .img-inch {
+    width: 30px;
+    height: 47px;
+  }
+
+  .data__figure {
+    font-size: 32px;
+  }
+
+  .data__ins {
+    font-size: 19px;
+  }
+
+  .img-wifi {
+    width: 38px;
+    height: 28px;
+  }
+
+  .data__wireless-update {
+    font-size: 18px;
+  }
+
+  .description__bag {
+    padding-bottom: 70px;
+  }
+
+  .img-cap {
+    width: 30px;
+    height: 44px; 
+  }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)